js/app/Config.js

```diff
@@ -7,8 +7,11 @@
  * @author Bernhard Posselt <dev@bernhard-posselt.com>
  * @copyright Bernhard Posselt 2014
  */
-app.config(
-    function ($routeProvider, $provide, $httpProvider, $locationProvider) {
+Config.$inject = ['$routeProvider', '$provide', '$httpProvider',
```

Comment:
That's of course one way to do it. The reason why this is needed is minification, apart from that its just needless repetition. Check out https://github.com/jeffling/ng-annotate-webpack-plugin (or similar plugins) that should do this for you automatically.